<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryView</class>
 <widget class="QWidget" name="QueryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>765</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="Input">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="executeQuery">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteQuery">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="edit-delete">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loadPreviousQuery">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="edit-undo">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loadNextQuery">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="edit-redo">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="newQuery">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>New Query</string>
     </property>
     <property name="icon">
      <iconset theme="document-new">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="queryLabel">
     <property name="text">
      <string>Query Variables</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="queryVariableList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: hsv(100, 30, 250)</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="conditionLabel">
     <property name="text">
      <string>Condition</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="conditionVariableList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: hsv(20, 30, 250)</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="interventionLabel">
     <property name="text">
      <string>Interventions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="interventionVariableList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: hsv(60, 30, 250)</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="edgeAdReLabel">
     <property name="text">
      <string>Edge Additions and Removals</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="edgeAdditionsRemovalList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="valueLabel">
       <property name="text">
        <string>Value Assignment:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="valueMaxLabel">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="probLabel">
       <property name="text">
        <string>Probability: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="probResultLabel">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Input</sender>
   <signal>returnPressed()</signal>
   <receiver>executeQuery</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
